link_test.go

```diff
@@ -0,0 +1,148 @@
+package toxiproxy
```

Comment:
Yep, you can add multiple of the same type, as long as they have a different name.
Tested here: https://github.com/Shopify/toxiproxy/pull/62/files#diff-095e965c1e92c9277036371d89bd083cR409